src/main/java/lotto/controller/InputValidator.java

```diff
@@ -0,0 +1,33 @@
+package lotto.controller;
+
+import static lotto.controller.ErrorMessages.INVALID_BONUS_NUMBER_FORMAT;
+import static lotto.controller.ErrorMessages.INVALID_PAID_AMOUNT_FORMAT;
+import static lotto.controller.ErrorMessages.INVALID_WINNING_NUMBERS_FORMAT;
+
+public class InputValidator {
+
+    private static final String REGEX_OF_POSITIVE_INTEGER = "^[1-9]\\d*$";
+    private static final String REGEX_OF_VALID_WINNING_LOTTO_FORMAT = "^[1-9]\\d*(,[1-9]\\d*)*$";
+
+    private InputValidator() {
+    }
+
+    public static void validatePaidAmount(String input) {
+        if (!input.matches(REGEX_OF_POSITIVE_INTEGER)) {
+            throw new IllegalArgumentException(INVALID_PAID_AMOUNT_FORMAT);
+        }
+    }
+
+    public static void validateWinningNumbers(String input) {
+        if (!input.matches(REGEX_OF_VALID_WINNING_LOTTO_FORMAT)) {
+            throw new IllegalArgumentException(INVALID_WINNING_NUMBERS_FORMAT);
+        }
+    }
+
+    public static void validateBonusNumber(String input) {
+        if (!input.matches(REGEX_OF_POSITIVE_INTEGER)) {
+            throw new IllegalArgumentException(INVALID_BONUS_NUMBER_FORMAT);
+        }
+    }
```

Comment:
if 문안에 부정문을 쓰는 것 보다 메서드를 분리하는 방법은 어떨까요!? 

```suggestion
    private boolean isNotMatchRegex(String input, String regex) {
         return input.matches(regex)
    }
   
    public static void validateBonusNumber(String input) {
        if (isNotMatchRegex(input , REGEX_OF_POSITIVE_INTEGER)) {
            throw new IllegalArgumentException(INVALID_BONUS_NUMBER_FORMAT);
        }
    }
```

Reply:
헉~! 이거 제가 항상 피하려고 하는 부정 연산자의 사용인데.. 이번에 놓쳤네요 ㅠ.ㅠ 감사합니다!